### ISSUE TITLE
Using multiple variables breaks rendering

### ISSUE THREAD

**Issue by Spaideri** [jameslnewell/styled-components-breakpoint#11]
When breakpoint string contains two variables like this:

`${breakpoint('tablet')`
    background: ${'grey'};
    color: ${'white'};
`}`
the string pattern start and end chars are missing because I don't know how to escape them in this editor 😃 

rendering breaks and results in combined value:
`background: greywhite;`

**Comment by omidfi** [comment]
```javascript
${breakpoint('tablet')`
background: ${'grey'};
color: ${'white'};
`}
```

**Comment by jameslnewell** [comment]
🤦‍♂️ doh!

Thanks! Fixed and released in `2.0.2`. Test added.

**Comment by omidfi** [comment]
That was quick. Thanks a  💯
